lib/realtime/tenants/repo/migrations/20240401105812_create_realtime_admin_and_move_ownership.ex

```diff
@@ -0,0 +1,36 @@
+defmodule Realtime.Tenants.Migrations.CreateRealtimeAdminAndMoveOwnership do
+  @moduledoc false
+
+  use Ecto.Migration
+
+  def change do
+    execute("CREATE ROLE supabase_realtime_admin WITH NOINHERIT NOLOGIN NOREPLICATION")
+
+    execute("GRANT ALL PRIVILEGES ON SCHEMA realtime TO supabase_realtime_admin")
+    execute("GRANT ALL PRIVILEGES ON ALL TABLES IN SCHEMA realtime TO supabase_realtime_admin")
+    execute("GRANT ALL PRIVILEGES ON ALL SEQUENCES IN SCHEMA realtime TO supabase_realtime_admin")
```

Comment:
```suggestion
    execute("GRANT ALL PRIVILEGES ON ALL SEQUENCES IN SCHEMA realtime TO supabase_realtime_admin")
    execute("GRANT ALL PRIVILEGES ON ALL FUNCTIONS IN SCHEMA realtime TO supabase_realtime_admin")

```

Are we missing a grant on functions here?